README.md

```diff
@@ -115,7 +115,7 @@ To use objective-c code from swift, you need to use bridging-header.
 ```swift
 import UIKit
 
-// ... in a view controller
+// ... in a view controller that inherits from MDCSwipeToChooseDelegate
```

Comment:
The view controller can't be *derived from* `MDCSwipeToChooseDelegate`, since that's a protocol. It should *conform to* that protocol.